regulations/views/preamble.py

```diff
@@ -47,14 +47,15 @@ def get(self, request, *args, **kwargs):
             raise Http404
 
         context = generate_html_tree(subtree)
-        template = context['node']['template_name']
         context['use_comments'] = True
+        template = context['node']['template_name']
 
+        context = {'sub_context': context, 'sub_template': template,
+                   'preamble': preamble}
         if not request.is_ajax() and request.GET.get('partial') != 'true':
-            # Wrap the inner context
-            context = {'sub_context': context, 'sub_template': template,
-                       'preamble': preamble}
             template = 'regulations/preamble-chrome.html'
+        else:
+            template = 'regulations/preamble-partial.html'
```

Comment:
One thing to keep in mind is that we may also need to trigger events in `sub-head-view.js` based on the design of the active titles that is unique n&c.